### DISCUSSION TITLE
Log dir for user instance services

### DISCUSSION THREAD

**Discussion by omnigenous** [davmac314/dinit#336] [Q&A] [answered]
What is appropriate directory to store services log files when running dinit as user instance?

For system instance we use `/var/log/dinit` but if `logfile` in user instance points to that dir dinit will, expectantly, return an error: `dinit: SERVICE: execution failed - opening log file: Permission denied`.

**Comment by mobin-2008** [comment, marked as answer]
Hello.

This is a system policy related question and we don't say much about system/distribution politics in dinit. I assume that you use Artix Linux, So Artix Linux forum (https://forum.artixlinux.org) is probably a better place for your question. 

Anyway you can store log files everywhere that you have permission to write at there, This means if system has not have any default path (has Artix?) you can create a directory in your system for user-services log files.

Best Regards

**Reply by omnigenous** [reply]
Ah I see, not right place to ask again, sorry :D

> Anyway you can store log files everywhere that you have permission to write at

Was just wondering what is a good practice so I don't regret my nooby decision in future.

**Reply by realroot2185** [reply]
But you need to make the folder right? Otherwise dinit will fail to start the service.

**Comment by omnigenous** [comment]
https://forum.artixlinux.org/index.php/topic,6918.new.html
